index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Embedding Images and Video</title>
+</head>
+
+<body>
+    <div class="container grid">
+        <header class=" grid">
+            <nav class="navbar">
+                <div class="icon">
+                  <span> </span>
+                  <span> </span>
+                  <span> </span>
+                  </div>
+                <div class="brand">
+                  <img src="https://archive.vn/Bss88/51a3832e6af6792add876216da444db23a4bea96.png" alt="brand">
+                </div>
+                <div class="search">
```

Comment:
- The search button should be to the left of the search bar.
![Annotation 2020-06-09 081954](https://user-images.githubusercontent.com/57726348/84100713-140d8480-aa2a-11ea-8c73-b6625b35775b.jpg)